make/devices.cmake

```diff
@@ -268,6 +279,45 @@ function(DEFINE_DEVICE_TYPE)
 
 endfunction()
 
+function(XML_LINT)
```

Comment:
This doesn't belong in device.cmake.  Maybe an xml.cmake?

Reply:
Done